src/battle/anim/ghost.c

```diff
@@ -189,3 +202,402 @@ const struct SpriteTemplate gSpriteTemplate_83DAF98 =
     .affineAnims = gDummySpriteAffineAnimTable,
     .callback = sub_80DF49C,
 };
+
+void sub_80DDB6C(struct Sprite *sprite) {
+    InitAnimSpritePos(sprite, 1);
+    sprite->data[0] = gBattleAnimArgs[2];
+    sprite->data[1] = sprite->pos1.x;
+    sprite->data[2] = GetBattlerSpriteCoord(gAnimBankTarget, 2);
+    sprite->data[3] = sprite->pos1.y;
+    sprite->data[4] = GetBattlerSpriteCoord(gAnimBankTarget, 3);
+    sub_8078BD4(sprite);
+    sprite->callback = sub_80DDBD8;
+    sprite->data[6] = 0x10;
+    REG_BLDCNT = 0x3F40;
```

Comment:
If you're feeling up to it, you can OR together the `BLDCNT`-related constants found here: https://github.com/pret/pokeruby/blob/master/include/gba/io_reg.h#L543